packages/client/src/index.test.ts

```diff
@@ -235,7 +261,8 @@ describe('LogtoClient', () => {
         storage,
       });
       await logto.loginWithRedirect(REDIRECT_URI, jest.fn());
-      await logto.handleCallback(REDIRECT_CALLBACK);
+      const { state } = new SessionManager(storage).get()!;
+      await logto.handleCallback(`${REDIRECT_CALLBACK}&state=${state}`);
```

Comment:
change `REDIRECT_CALLBACK` directly? seen many times.

Reply:
`state` is a random value, so i have to get it from session.

Reply:
TBD.

I extract `return new SessionManager(storage).get()!.state;` to a method? @wangsijie

Reply:
can we use a fake state string such as `fake_state` here?
since `generate random state value` is been tested elsewhere

```
const state = 'state';
```

Reply:
Fixed.